modules/ROOT/pages/dataweave-types.adoc

```diff
@@ -253,7 +253,76 @@ var myDate = |2003-10-01T23:57:59.700-03:00|
 [[date_format_change]]
 === Changing the Format of a Date
 
-You can specify a date to be in any format you prefer through using *as* in the following way:
+
+To format dates and times, DataWeave supports some formatting characters 
+that are based on the Java 8 `java.time` package, such as the `y`, `M`, 
+and `d` in the date format `yyyy-MM-dd`. The following example formats 
+the output of the `now` DataWeave function to show supported letters:
+
+.DataWeave Script
+[source,dataweave,linenums]
+----
+%dw 2.0
+output application/json
+---
+[
+  { "dateTime" : now() },
+  { "era" : now() as String { format: "G"} }, 
+  { "year" : now() as String { format: "y"} },
+// Y (for week year) is not supported.
+  { notDayNumberOfWeek: now() as String {format: "u"} },
+  { "monthInYearContextSensitive": now() as String { format: "M"} }, 
+  { "monthInYearStandalone" : now() as String { format: "L"} },
+  { "weekInYear" : now() as String {format: "w"} },
+  { "weekInMonth" : now() as String {format: "W"} }, 
+  { "dayOfMonth" : now() as String {format: "d"} }, 
+  { "dayOfWeekInMonth" : now() as String {format: "F"} },
+  { "dayNameAbbreviated" : now() as String {format: "E"} },
+// u (for day number of the week) is not supported.
```

Comment:
`u` is proleptic year
See https://stackoverflow.com/questions/29014225/what-is-the-difference-between-year-and-year-of-era
for more info